### ISSUE TITLE
Meet with Milad for onboarding of VZ technologies (local stack + polygon tooling)

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#12391]
Milad has joined the VZ team. Milad scheduled a 1:1 with @frankhereford from 9AM - 12PM on June 1st. This issue is to track the work of a 1:1 meeting with him to make sure he has the following technologies and capabilities available to him:

* [x] VZ local stack
  * [x] In particular the VZDB, with replication working
* [x] Polygonizer tooling, likely best approached though a docker containerization of that work
* [x] QGIS setup with access to:
  * [x] Local DB in a RW configuration
  * [x] Production DB in a RO configuration via the RR
  * [x] Frank's OSM tiling
  * [x] Frank's NearMap proxy container
* [ ] Get ZenHub up and working on Milad's computer

In meeting actions:
* [x] Docker Desktop
* [x] GitHub Desktop
* [x] QGIS 3.30.0

Preparation work required:
* [x] Investigate and dockerize the polygonizer in as much as it's useful; at least producing a VM-like container where the code is checked out and can be ran, provided the operator collects the required input data.

**Comment by patrickm02L** [comment]
> @patrickm02L Here's the issue re my meeting with Milad. I'm not 100% that this is sized well - what do you think it should be?

You're estimate works!

**Comment by frankhereford** [comment]
@amenity I met with Milad today, and I think that there may be something up with his zenhub invitation. He's got the plugin installed on his desktop and that all looks good. I checked the zenhub user list and I didn't see his name on there. I'm not sure what the next step is, but I did want to let you know. Thanks!

**Comment by frankhereford** [comment]
This was a great meeting, and we covered everything we had hoped to. We were able to get the local VZ DB running, but ran into some complications with his local python installation, so we were not able to use the `vision-zero` tooling. We worked around this by extracting the `docker compose` commands from the script, and we got him those that he can use to replicate and populate the local database. This will give him a workable copy of the database which he can use to experiment and develop his own processes and to do more complex geospatial analysis. Additionally, we installed QGIS so that he can visualize the geometry stored within the database. 

I am in progress of publishing the new QGIS plugin for polygonization, and when that is up on the QGIS plugin repository, I'll let him know so he can install that in the normal way. 

I let him know also that I'm available on slack, email and a potential second meeting to review the material we covered today. It was a lot of commands to be used in a terminal context, and I'm very willing to help advise on how to plug those together to achieve what he's looking to do.
